staging/src/k8s.io/code-generator/generate-groups.sh

```diff
@@ -77,13 +77,25 @@ if [ "${GENS}" = "all" ] || grep -qw "deepcopy" <<<"${GENS}"; then
       "$@"
 fi
 
+if [ "${GENS}" = "all" ] || grep -qw "applyconfiguration" <<<"${GENS}"; then
+  echo "Generating apply configuration for ${GROUPS_WITH_VERSIONS} at ${OUTPUT_PKG}/${APPLYCONFIGURATION_PKG_NAME:-applyconfiguration}"
+  "${gobin}/applyconfiguration-gen" \
+      --input-dirs "$(codegen::join , "${FQ_APIS[@]}")" \
+      --output-package "${OUTPUT_PKG}/${APPLYCONFIGURATION_PKG_NAME:-applyconfiguration}" \
+      "$@"
+fi
+
 if [ "${GENS}" = "all" ] || grep -qw "client" <<<"${GENS}"; then
   echo "Generating clientset for ${GROUPS_WITH_VERSIONS} at ${OUTPUT_PKG}/${CLIENTSET_PKG_NAME:-clientset}"
+  if [ "${GENS}" = "all" ] || grep -qw "applyconfiguration" <<<"${GENS}"; then
+    APPLY_CONFIGURATION_PACKAGE="${OUTPUT_PKG}/${APPLYCONFIGURATION_PKG_NAME:-applyconfiguration}"
+  fi
   "${gobin}/client-gen" \
       --clientset-name "${CLIENTSET_NAME_VERSIONED:-versioned}" \
       --input-base "" \
       --input "$(codegen::join , "${FQ_APIS[@]}")" \
       --output-package "${OUTPUT_PKG}/${CLIENTSET_PKG_NAME:-clientset}" \
+      --apply-configuration-package "${APPLY_CONFIGURATION_PACKAGE:-}" \
```

Comment:
```suggestion
      --apply-configuration-package "${APPLY_CONFIGURATION_PACKAGE:-applyconfiguration}" \
```

Here too?

Reply:
That one is a bit different from the others above. It corresponds to the package of apply configurations, previously generated by `applyconfiguration-gen`, and that's optional so Apply functions are not generated if empty, which happens if the `applyconfiguration` generator wasn't specified.